charts/opentelemetry-operator/templates/admission-webhooks/mutatingwebhookconfiguration.yaml

```
@@ -0,0 +1,51 @@
+{{- if and .Values.admissionWebhooks.enabled }}
```

```suggestion
{{- if .Values.admissionWebhooks.enabled }}
```

This seems out of place since there's only one operand.

---

Yes, this is my mistake. Thank you for pointing out.